docs/index.asciidoc

```diff
@@ -87,6 +88,16 @@ filter {
   }
 }
 
+[id="plugins-{type}s-{plugin}-parser_options"]
+===== `parser_options`
+
+* Value type is <<string,string>>
+
+Setting XML parser options allows for more control of the parsing process.
+By default the parser is not strict and thus accepts some invalid content.
+Currently supported options are:
+
+ - _strict_ - forces the parser to fail early when content is not valid xml
```

Comment:
```suggestion
 - `strict` - forces the parser to fail early when content is not valid xml
```

Reply:
>"forces the parser to fail early when content is not valid xml"

Maybe include the alternative.  Something like "forces the parser to fail early instead of accumulating errors when content is not valid xml."